tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/Test_HalfSpaceMirror.cpp

```diff
@@ -140,3 +141,18 @@ SPECTRE_TEST_CASE(
         });
   };
 }
+
+// [[OutputRegex, The numerical integral in the AnalyticSolution fails]]
+SPECTRE_TEST_CASE("Unit.AnalyticSolutions.Elasticity.HalfSpaceMirror.Error",
+                  "[PointwiseFunctions][Unit][Elasticity]") {
+  ERROR_TEST();
+  const size_t dim = Elasticity::Solutions::HalfSpaceMirror::volume_dim;
+  const Elasticity::ConstitutiveRelations::IsotropicHomogeneous<dim>
+      constitutive_relation{36.36363636363637, 30.76923076923077};
+  const HalfSpaceMirrorProxy solution{0.177, constitutive_relation, 5, 1e-15,
+                                      1e-15};
```

Comment:
Make the tolerances exactly `0.`, then this will reliably fail

Reply:
GslQuad does not allow for both tolerances to be 0. at the same time.

Reply:
That makes sense I guess :) Keep as-is then